src/pages/LandingPage.tsx

```diff
@@ -78,22 +90,51 @@ export const LandingPage: FunctionComponent = () => {
 
             <PageSection variant={PageSectionVariants.light} className="pf-u-p-md">
               <div className="pf-u-text-align-right">
-                <Button variant="link" icon={<ThIcon />} className="pf-u-mr-sm" isInline isLarge/>
-                <Button variant="link" icon={<ThListIcon />} isInline isLarge isDisabled/>
+                <Button variant="link" icon={<ThIcon />} onClick={() => setView('grid')} className="pf-u-mr-sm" isInline isLarge/>
+                <Button variant="link" icon={<ThListIcon />} onClick={() => setView('list')} isInline isLarge/>
               </div>
             </PageSection>
 
             <PageSection className="apid-c-page__main-section-gallery" style={paginatedGalleryInfo.height ? {minHeight: `${paginatedGalleryInfo.height}px !important`} : undefined} padding={{ default: 'noPadding' }} isFilled={true}>
-              <GalleryTemplate
-                  id={galleryId}
-                  elements={filteredDocs}
-                  isHidden
-              />
-              { paginatedGalleryInfo.paginatedElements.length > 0 ?
+            { view === 'grid'
+              ? <>
                   <GalleryTemplate
-                      elements={paginatedGalleryInfo.paginatedElements}
-                  /> :
-              <NoMatchFound clearFilters={clearFilters} /> }
+                    id={galleryId}
+                    elements={filteredDocs}
+                    isHidden
+                  />
+                  { paginatedGalleryInfo.paginatedElements.length > 0 ?
+                    <GalleryTemplate
+                        elements={paginatedGalleryInfo.paginatedElements}
+                    /> : <NoMatchFound clearFilters={clearFilters} /> }
+                </>
+              : <>
+
+                  <TableComposable aria-label="Misc table">
+                  <Thead noWrap>
+                    <Tr>
+                      <Th>
+                        <Flex>
+                          <FlexItem>
+                            {columnNames.name}
+                          </FlexItem>
+                        </Flex>
+                      </Th>
+                      <Th>{columnNames.description}</Th>
+                      <Th>{columnNames.apiVersion}</Th>
```

Comment:
Since we don't have the api version info yet, instead of saying `placeholder` we can probably omit this column altogether for now.